b/beanshell/beanshell_ubi_8.4.sh

```diff
@@ -0,0 +1,62 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : BeanShell
+# Version       : 2.0b6
+# Source repo   : https://github.com/beanshell/beanshell
+# Tested on     : UBI: 8.4
+# Script License: Apache License 2.0
+# Maintainer's  : Sapana Khemkar <Sapana.Khemkar@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+set -e
+
+# Variables
+PACKAGE_NAME=beanshell
+PACKAGE_URL=https://github.com/beanshell/beanshell.git
+PACKAGE_VERSION=2.0b6
+
+
+# install tools and dependent packages
+yum install -y git wget
+
+# install java
+yum -y install java-1.8.0-openjdk-devel
+
+#install maven
+#cd /opt/
+#wget https://www-eu.apache.org/dist/maven/maven-3/3.8.4/binaries/apache-maven-3.8.4-bin.tar.gz
+#tar xzf apache-maven-3.8.4-bin.tar.gz
+#ln -s apache-maven-3.8.4 maven
+#export MVN_HOME=/opt/maven
+#export PATH=${MVN_HOME}/bin:${PATH}
+#mvn -version
```

Comment:
Can you remove the commented lines (if they are not kept on purpose) ?

Reply:
removed as it is not required at anymore